src/main/scala/org/virtuslab/akkasaferserializer/StandardCodecs.scala

```diff
@@ -0,0 +1,30 @@
+package org.virtuslab.akkasaferserializer
+
+import io.bullet.borer.Codec
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
+import java.time.OffsetDateTime
+
+object StandardCodecs {
+  implicit val offsetDateTimeCodec: Codec[OffsetDateTime] =
+    Codec.bimap[Array[Byte], OffsetDateTime](
+      serializeSerializable[OffsetDateTime],
+      deserializeSerializable[OffsetDateTime]
+    )
+
+
+  private def serializeSerializable[T <: java.io.Serializable](ser: T): Array[Byte] = {
+    val os = new ByteArrayOutputStream()
+    val oos = new ObjectOutputStream(os)
+    oos.writeObject(ser)
```

Comment:
Hmm this uses Java serialization under the hood right? This is what we'd like to avoid, mostly due to performance & security concerns: https://doc.akka.io/docs/akka/current/serialization.html#java-serialization

Reply:
It's okay just for now for prototyping, but this needs to be replaced... pls take a look at https://github.com/plokhotnyuk/jsoniter-scala, this might be useful both OffsetDateTime and for #7

Reply:
(so: pls leave as is on this PR + open an issue for better implementation)